scene/gui/tree.cpp

```diff
@@ -2228,10 +2228,18 @@ int Tree::draw_item(const Point2i &p_pos, const Point2 &p_draw_ofs, const Size2
 					}
 
 					if (p_item->cells[0].selected) {
-						if (has_focus()) {
-							theme_cache.selected_focus->draw(ci, row_rect);
+						if (drop_mode_flags || !is_row_hovered) {
```

Comment:
What's `drop_mode_flags` for?

Reply:
I think it's just a leftover from a code change. I'll check and correct it as soon as I get back from my trip on the 04. Jan.

Reply:
Removed it.